app/graphql/mutations/dropout_student.rb

```diff
@@ -0,0 +1,22 @@
+module Mutations
+  class DropoutStudent < GraphQL::Schema::Mutation
+    argument :id, ID, required: true
+
+    description "Mark student as exited"
+
+    field :success, Boolean, null: false
+
+    def resolve(params)
+      mutator = DropoutStudentMutator.new(context, params)
+
+      if mutator.valid?
+        mutator.execute
+        mutator.notify(:success, 'Student updated successfully', 'Reloading page')
```

Comment:
@bodhish : Refer to https://github.com/SVdotCO/pupilfirst/issues/126 checklist. The user needs to be notified that the student is archived. Also, rename _Inactive Students_ to _Student Archive_ as mentioned in the checklist.